operations/orders.md

```diff
@@ -77,15 +77,6 @@ Creates a new order, with the specified products and items. If the time of consu
 | `UnitAmount` | [Amount parameters](#amount-parameters) | required | Unit amount, e.g. amount for one beer \(note that total amount of the item is therefore `UnitAmount` times `UnitCount`\). |
```

Comment:
```suggestion
| `UnitAmount` | [Amount parameters](_objects.md#amount-parameters) | required | Unit amount, e.g. amount for one beer \(note that total amount of the item is therefore `UnitAmount` times `UnitCount`\). |
```

Reply:
There is one other instance of `#amount-parameters` on this page that also needs updating to point to `_objects.md`.